README.rst

```diff
@@ -2842,8 +2911,9 @@ error If the request does not succeed, it will contain a dictionary with an erro
 
 Local Models
 ------------
-If you want to use a specfic connection object for the remote retrieval, you can set it as second parameter::
-    require 'vendor/autoload.php';
+
```

Comment:
I would explain what a local model is and how it works before explaining that you can specify the connection object.
Maybe something like:

You can use the information returned by the API when asking for a model to create a Model object in your own computer that will be able to produce predictions with no further connection to the remote API. The local Model object can be instantiated by using the entire response of the GET call to the API (first example), but it also accepts the model ID as first argument. In this case, a new connection will be created internally to download the model information. If you want to...

Reply:
Excellent, thank you, I will make the change.